zcncore/smartcontract_executor_mobile.go

```diff
@@ -11,7 +11,7 @@ type SmartContractExecutor interface {
 }
 
 func (t *Transaction) ExecuteSmartContract(address, methodName string, input string, val uint64) error {
-	err := t.createSmartContractTxn(address, methodName, json.RawMessage(input), val)
+	err := t.createSmartContractTxn(address, methodName, json.RawMessage([]byte(input)), val)
```

Comment:
Seems  the []byte convert is unnecessary, the result is the same

Reply:
reverted